src/Symfony/Bundle/FrameworkBundle/Test/TestContainer.php

```diff
@@ -17,6 +17,9 @@
 use Symfony\Component\HttpKernel\KernelInterface;
 
 /**
+ * A special container used in tests. This gives access to both public and
+ * private services.
```

Comment:
I think it's important to indicate that it does *not* include removed private services (e.g. those that are inlined or not used at all).

Reply:
Thank you